lib/seed/Seeder.js

```diff
@@ -41,22 +44,25 @@ class Seeder {
   async _listAll(config) {
     this.config = this.setConfig(config);
     const loadExtensions = this.config.loadExtensions;
-    return readdir(this._absoluteConfigDir()).then((seeds) =>
-      seeds
-        .filter((value) => {
-          const extension = path.extname(value);
-          return includes(loadExtensions, extension);
-        })
-        .sort()
-    );
+    const seeds = (
```

Comment:
Could you please replicate sortDirsSeparately parameter from migrations for consistency? (https://github.com/knex/knex/blob/fa906278b823728f02349be7315d536de6feb7b1/lib/migrate/sources/fs-migrations.js#L47)

To provide context: this would allow users to either run seeds directory by directory (in the order specified in configuration), or have global execution order (all filenames sorted irrespective of the directory they are located in)

Reply:
I've implemented the `sortDirsSeparately`. Tell me if it sounds ok for you.